maven-embedder/src/main/java/org/apache/maven/cli/transfer/ConsoleMavenTransferListener.java

```diff
@@ -35,7 +35,7 @@
  */
 public class ConsoleMavenTransferListener extends AbstractMavenTransferListener {
 
-    private Map<TransferResource, Long> transfers = new LinkedHashMap<>();
+    private Map<TransferResourceIdentifier, TransferResourceAndSize> transfers = new LinkedHashMap<>();
```

Comment:
Those 3 fields should be `final`.

Reply:
Done in https://github.com/apache/maven/pull/1575/commits/274f1807ab0fcef625efadb26aabcaa2be1fe31b